modules/cnf-topology-aware-lifecycle-operator-installation-web-console.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
```

Comment:
Again, will have to double check these are the exact right steps once we have TALO on OperatorHub.